lib/exercism/user_contribution.rb

```diff
@@ -0,0 +1,64 @@
+require_relative '../../x'
+class UserContribution
+  attr_reader :user
+  def initialize(user)
+    @user = user
+  end
+
+  # Return Hahs with, name of the type of contribution, also the erb partial and
+  # the info related with each contributions
+  def list
+    @list ||= { completed_tracks: completed_tracks }
```

Comment:
If the class keep growing I tend to move the logic to separate class, this will help test and adding future features, but for now I don't know if I'm going to need this methods, for the other type of contributions.

In this example I will move the `completed_tracks` to it's own class like `CompletedTracks.call(user)` and all the methods related